lib/cli/parse.rs

```diff
@@ -671,7 +734,7 @@ pub(super) fn global_state_identifier(
 }
 
 /// `purse_id` can be a formatted public key, account hash, or URef.  It may not be empty.
-pub(super) fn purse_identifier(purse_id: &str) -> Result<PurseIdentifier, CliError> {
+pub fn purse_identifier(purse_id: &str) -> Result<PurseIdentifier, CliError> {
```

Comment:
why are we making the identifier parsers public?

Reply:
to be able to use it outside in query_balance and query_balance_details in the SDK with so that we avoid duplicating this parser helper like on dev account_identifier / transfer_target / args_from_simple_or_json / uref those helpers are very convenient to consume outside instead of recoding them again with same code 

```
use casper_client::cli::parse::purse_identifier as parse_purse_identifier;
```

Reply:
ok